### PR TITLE
(feature) Scala: add link to v2 devfile

### PR DESCRIPTION
**Pull request by sideeffffect**
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: devfiles/scala-sbt/meta.yaml [hunk 1]
```diff
@@ -3,3 +3,5 @@ displayName: Scala
 description: Scala Stack with OpenJDK 11 and sbt 1.x
 tags: ["Scala", "OpenJDK", "sbt", "Debian"]
 icon: /images/scala.svg
+links:
+  v2: https://github.com/sideeffffect/console-scala-simple/tree/patch-1
```
Comment (svor) [inline devfiles/scala-sbt/meta.yaml:7]:
```suggestion
  v2: https://github.com/che-samples/scala-sbt/tree/devfilev2
```

Comment (svor) [reply devfiles/scala-sbt/meta.yaml:7]: @sideeffffect let's update the link to use devfile form the che-samples

Comment (sideeffffect) [reply devfiles/scala-sbt/meta.yaml]: Thanks

**Review by svor — APPROVED** [review]
@sideeffffect Thanks!

**Pull request merged**